### PR TITLE
Install fits as prerequisite

### PR DESCRIPTION
**Pull request by scollazo**
Closes artefactual-labs/am-packbuild#111

### CODE REVIEW

File: admin-manual/installation/installation.rst [hunk 1]
```diff
@@ -316,10 +316,14 @@ place to store packages for multiple operating systems. Packages for both Ubuntu
    use 'demo' as the password during the install process. The password can be
    changed after the installation is complete.
 
+   The fits package need to be installed as a prerequesite, due to a circular dependencies
+   problem that is exclusive of Ubuntu 14.04.
+
```
Comment (jrwdunham) [inline admin-manual/installation/installation.rst]: Nitpick: the wording "that is exclusive of Ubuntu 14.04" sounds a bit odd to me. I think one of the following would be better:

- only happens on Ubuntu 14.04
- arises only on Ubuntu 14.04
- occurs exclusively on Ubuntu 14.04

**Comment by scollazo** [comment]
@jrwdunham ping

File: admin-manual/installation/installation.rst [hunk 2]
```diff
@@ -316,10 +316,14 @@ place to store packages for multiple operating systems. Packages for both Ubuntu
    use 'demo' as the password during the install process. The password can be
    changed after the installation is complete.
 
+   The fits package need to be installed as a prerequesite, due to a circular dependencies
+   problem that arises only on Ubuntu 14.04.
```
Comment (jrwdunham) [inline admin-manual/installation/installation.rst]: Typos: change `need` to `needs` and `prerequesite` to `prerequisite`.

**Review by jrwdunham — APPROVED** [review]
LGTM. Just fix a couple of typos.

**Pull request merged**